src/structure/structure.ts

```diff
@@ -0,0 +1,26 @@
+import { apiClient } from '../erudio';
+
+export interface Options {
+  filter?: string;
+  page?: number;
+  limit?: number;
+  sort?: string;
+}
+
+export class Structure {
+  private base_url: string;
+
+  constructor(base_url: string) {
+    this.base_url = base_url;
+  }
+
+  public async listNodes(namespace: string, options?: Options): Promise<any> {
+    const url = this.base_url + `/structures/${namespace}/nodes/`;
+    try {
+      const res = await apiClient.get(url, { params: options });
+      return res.data;
+    } catch (error) {
+      console.error(`${error.response.path} -> ${error.response.status}`);
```

Comment:
@JERILJOSE96 error handling should be propagated to clients of this library, let them decide what to do with errors